lib/developer_portal/app/controllers/developer_portal/admin/account/personal_details_controller.rb

```diff
@@ -10,25 +10,21 @@ class DeveloperPortal::Admin::Account::PersonalDetailsController < ::DeveloperPo
 
   liquify prefix: 'user'
 
+  before_action :verify_current_password, only: :update
+
   def show
     assign_user_drop
   end
 
   def update
-    #TODO: write tests for this
     resource.validate_fields!
-    update! do |success, failure|
-      success.html do
-        if resource.just_changed_password?
-          resource.kill_user_sessions(user_session)
-        end
-        redirect_to(redirect_path)
-      end
+    if resource.errors.empty? && resource.update(user_params)
+      resource.kill_user_sessions(user_session) if resource.just_changed_password?
 
-      failure.html do
-        assign_user_drop
-        render :action => 'show'
-      end
+      redirect_to redirect_path, notice: 'User was succesfully updated.'
```

Comment:
`succesfully` => `successfully`

I think this typo was already in the provider controllers, so maybe we need to fix it there also.

Reply:
Nice catch! 🎯